Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -155,7 +155,8 @@ DECLARE_SOA_COLUMN(FlagMCMatchRec, flagMCMatchRec, int8_t); // reconstruction le
 DECLARE_SOA_COLUMN(FlagMCMatchGen, flagMCMatchGen, int8_t); // generator level
 
 // mapping of decay types
-enum DecayType { D0ToPiK = 1 };
+enum DecayType { D0ToPiK = 1,
+                 JpsiToEE = 1 };
```

Comment:
This is wrong. `JpsiToEE` must not be equal to `D0ToPiK`.
`JpsiToEE = 1` must be replaced with `JpsiToEE`.

Reply:
> This is wrong. `JpsiToEE` must not be equal to `D0ToPiK`.
> `JpsiToEE = 1` must be replaced with `JpsiToEE`.

Hi Vit, thanks for reminding me here. I will make a commit to replace it.